### PR TITLE
Add `technical` attribute to text containing elements.

### PR DESCRIPTION
**Pull request by augb**
Closes openlawlibrary/OpenLawPlatform#14.

### CODE REVIEW

File: schemas/dc-library.xsd [hunk 1]
```diff
@@ -94,6 +97,7 @@
       <xs:simpleContent>
         <xs:extension base="xs:string">
           <xs:attribute name="root" type="xs:string" use="optional"/>
+          <xs:attribute name="technical" type="xs:string" use="optional"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: should not be available in metadata

File: schemas/dc-library.xsd [hunk 2]
```diff
@@ -210,7 +214,7 @@
   <xs:element name="emergency">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="law" type="xs:string"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: ditto

File: schemas/dc-library.xsd [hunk 3]
```diff
@@ -219,7 +223,7 @@
   <xs:element name="federal">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="law" type="xs:string"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: ditto

File: schemas/dc-library.xsd [hunk 4]
```diff
@@ -240,6 +245,7 @@
     <xs:complexType>
       <xs:complexContent mixed="true">
         <xs:extension base="xs:anyType">
+          <xs:attribute name="technical" type="xs:string" use="optional"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: ditto

File: schemas/dc-library.xsd [hunk 5]
```diff
@@ -271,7 +277,7 @@
   <xs:element name="legislation">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="committeeReferral" type="xs:string"/>
+        <xs:element name="committeeReferral" type="technicalString"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: ditto

File: schemas/dc-library.xsd [hunk 6]
```diff
@@ -300,7 +306,7 @@
   <xs:element name="meta">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="effective" type="xs:string" minOccurs="0"/>
+        <xs:element name="effective" type="technicalString" minOccurs="0"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: ditto

File: schemas/dc-library.xsd [hunk 7]
```diff
@@ -419,7 +434,7 @@
   <xs:element name="votingSummary">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="readingDescription" type="xs:string"/>
+        <xs:element name="readingDescription" type="technicalString"/>
```
Comment (dgreisen) [inline schemas/dc-library.xsd]: ditto

File: schemas/dc-library.xsd (hunk 2 above)
Comment (augb) [inline schemas/dc-library.xsd]: done

File: schemas/dc-library.xsd (hunk 3 above)
Comment (augb) [inline schemas/dc-library.xsd]: done

File: schemas/dc-library.xsd (hunk 4 above)
Comment (augb) [inline schemas/dc-library.xsd]: done

File: schemas/dc-library.xsd (hunk 5 above)
Comment (augb) [inline schemas/dc-library.xsd]: done

File: schemas/dc-library.xsd (hunk 6 above)
Comment (augb) [inline schemas/dc-library.xsd]: done

File: schemas/dc-library.xsd (hunk 7 above)
Comment (augb) [inline schemas/dc-library.xsd]: done

File: schemas/dc-library.xsd (hunk 1 above)
Comment (augb) [inline schemas/dc-library.xsd]: I made the change in `meta`, my understanding is that `cite` still needs it. Is this correct?

Comment (dgreisen) [inline schemas/dc-library.xsd]: Correct. I tagged the wrong one - citation, which is inside meta - should not allow technical, but cite and code-cite should. It looks like citation still has it.

Comment (augb) [inline schemas/dc-library.xsd]: Gotcha. Removed from `<citation>`.

**Comment by augb** [comment]
Rebased off of `development` as of a few moments ago.

**Pull request closed without merging**